README.md

```diff
@@ -303,6 +303,20 @@ int main(){
 }
 ```
 
+By default we use:
+```C
+static roaring_memory_t global_memory_hook = {
+    .malloc = malloc,
+    .realloc = realloc,
+    .calloc = calloc,
+    .free = free,
+    .aligned_malloc = roaring_bitmap_aligned_malloc,
+    .aligned_free = roaring_bitmap_aligned_free,
+};
+```
+
+We use the C convention regarding `free`: `free(NULL)` should have no effect. Thus our code does not check whether the provided pointer is non-null.
```

Comment:
I think I'd word this as something like "we require the .free/.aligned_free function to follow the C convention where free(NULL) has no effect", stating it as a requirement vs saying what we do and having the user have to figure out what it means they need to do.

Reply:
Wording updated.